ginga/rv/plugins/ChangeHistory.py

```diff
@@ -300,8 +298,10 @@ def __str__(self):
         return 'changehistory'
 
 
-# Replace module docstring with config doc for auto insert by Sphinx.
-# In the future, if we need the real docstring, we can append instead of
-# overwrite.
+# Append module docstring with config doc for auto insert by Sphinx.
 from ginga.util.toolbox import generate_cfg_example  # noqa
-__doc__ = generate_cfg_example('plugin_ChangeHistory', package='ginga')
+if __doc__ is not None:
+    __doc__ += generate_cfg_example('plugin_ChangeHistory', package='ginga')
+
+# Need this for offline WBrowser doc
+ChangeHistory.__doc__ = __doc__
```

Comment:
I just realized that this is not Python 2 compatible. I can easily skip this code if Python 2 is detected, however in that case the "offline" doc will not show anything useful. So the question here for @ejeschke is -- Do we really need the *offline mode* to work in Python 2 given that Astropy is stopping Python 2 support in its v3 release and Ginga is an Astropy-affiliated package?